proxy/http/HttpSM.cc

```diff
@@ -7618,6 +7616,8 @@ HttpSM::do_redirect()
     return;
   }
 
+  redirection_tries++;
+
```

Comment:
hmm, looks like better to do redirection_tries++ when setting redirect_in_process = true, if you consider a really redirection. good catch.